README
NAME
    Task::BeLike::hanekomu - Install modules I like

VERSION
    version 1.103460

DESCRIPTION
    This Task installs modules that I need to work with. They are listed in
    this distribution's "Makefile.PL".

TASK CONTENTS
  Toolchain
   App::Ack
   App::Rgit
   Devel::NYTProf
   Devel::Loaded
   Devel::SearchINC
   Perl::Tidy
   Pod::Wordlist::hanekomu
   Dist::Zilla
   Dist::Zilla::PluginBundle::MARCEL
   Pod::Weaver::PluginBundle::MARCEL
   Dist::Zilla::Plugin::Git::Init
   App::cpanoutdated
   App::cpanminus
   App::distfind
   Pod::Coverage::TrustPod
  Useful modules
   YAML
   DBI
   DBD::SQLite
   File::Which
   File::Slurp
   Test::Differences
  Web, Networking, Events
   LWP
   Web::Scraper
   Coro
   AnyEvent
   Task::Plack
AUTHOR
    Marcel Gruenauer <marcel@cpan.org>

COPYRIGHT AND LICENSE
    This software is copyright (c) 2009 by Marcel Gruenauer.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.

INSTALLATION
    See perlmodinstall for information and options on installing Perl
    modules.

BUGS AND LIMITATIONS
    No bugs have been reported.

    Please report any bugs or feature requests through the web interface at
    <http://rt.cpan.org/Public/Dist/Display.html?Name=Task-BeLike-hanekomu>.

AVAILABILITY
    The latest version of this module is available from the Comprehensive
    Perl Archive Network (CPAN). Visit <http://www.perl.com/CPAN/> to find a
    CPAN site near you, or see
    <http://search.cpan.org/dist/Task-BeLike-hanekomu/>.

    The development version lives at
    <http://github.com/hanekomu/Task-BeLike-hanekomu.git> and may be cloned
    from <git://github.com/hanekomu/Task-BeLike-hanekomu.git>. Instead of
    sending patches, please fork this project using the standard git and
    github infrastructure.



Changes
Revision history for Perl extension Task-BeLike-hanekomu

1.103460  2010-12-12 12:37:44 Europe/Vienna
    - added Dist::Zilla::Plugin::Git::Init
    - added Pod::Coverage::TrustPod
    - instead of Task::Dist::Zilla, just require Dist::Zilla. The plugins I
      need will be brought in via Dist::Zilla::PluginBundle::MARCEL

1.101400  2010-05-20 14:06:31 Europe/Vienna
    - added App::distfind, changed from Plack to Task::Plack

1.100810  2010-03-22 16:49:12 Europe/Vienna
    - converted the distribution to Dist::Zilla-style

0.02  Sat 2010.02.06 15:32:32 CET (Marcel Gruenauer <marcel@cpan.org>)
    - added more modules that I like

0.01  2009-10-04T10:03:05Z (Marcel Gruenauer <marcel@cpan.org>)
    - original version


lib/Task/BeLike/hanekomu.pm
use 5.008;
use strict;
use warnings;

package Task::BeLike::hanekomu;
BEGIN {
  $Task::BeLike::hanekomu::VERSION = '1.103460';
}
# ABSTRACT: Install modules I like
1;


__END__
=pod

=head1 NAME

Task::BeLike::hanekomu - Install modules I like

=head1 VERSION

version 1.103460

=head1 DESCRIPTION

This L<Task> installs modules that I need to work with. They are listed in
this distribution's C<Makefile.PL>.

=head1 TASK CONTENTS

=head2 Toolchain

=head3 L<App::Ack>

=head3 L<App::Rgit>

=head3 L<Devel::NYTProf>

=head3 L<Devel::Loaded>

=head3 L<Devel::SearchINC>

=head3 L<Perl::Tidy>

=head3 L<Pod::Wordlist::hanekomu>

=head3 L<Dist::Zilla>

=head3 L<Dist::Zilla::PluginBundle::MARCEL>

=head3 L<Pod::Weaver::PluginBundle::MARCEL>

=head3 L<Dist::Zilla::Plugin::Git::Init>

=head3 L<App::cpanoutdated>

=head3 L<App::cpanminus>

=head3 L<App::distfind>

=head3 L<Pod::Coverage::TrustPod>

=head2 Useful modules

=head3 L<YAML>

=head3 L<DBI>

=head3 L<DBD::SQLite>

=head3 L<File::Which>

=head3 L<File::Slurp>

=head3 L<Test::Differences>

=head2 Web, Networking, Events

=head3 L<LWP>

=head3 L<Web::Scraper>

=head3 L<Coro>

=head3 L<AnyEvent>

=head3 L<Task::Plack>

=head1 AUTHOR

Marcel Gruenauer <marcel@cpan.org>

=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2009 by Marcel Gruenauer.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.

=head1 INSTALLATION

See perlmodinstall for information and options on installing Perl modules.

=head1 BUGS AND LIMITATIONS

No bugs have been reported.

Please report any bugs or feature requests through the web interface at
L<http://rt.cpan.org/Public/Dist/Display.html?Name=Task-BeLike-hanekomu>.

=head1 AVAILABILITY

The latest version of this module is available from the Comprehensive Perl
Archive Network (CPAN). Visit L<http://www.perl.com/CPAN/> to find a CPAN
site near you, or see L<http://search.cpan.org/dist/Task-BeLike-hanekomu/>.

The development version lives at L<http://github.com/hanekomu/Task-BeLike-hanekomu.git>
and may be cloned from L<git://github.com/hanekomu/Task-BeLike-hanekomu.git>.
Instead of sending patches, please fork this project using the standard
git and github infrastructure.

=cut



Makefile.PL

use strict;
use warnings;

BEGIN { require 5.008; }

use ExtUtils::MakeMaker 6.31;



my %WriteMakefileArgs = (
  'ABSTRACT' => 'Install modules I like',
  'AUTHOR' => 'Marcel Gruenauer <marcel@cpan.org>',
  'BUILD_REQUIRES' => {
    'Carp' => '0',
    'English' => '0',
    'File::Find' => '0',
    'File::Temp' => '0',
    'Scalar::Util' => '0',
    'Test::More' => '0.94'
  },
  'CONFIGURE_REQUIRES' => {
    'ExtUtils::MakeMaker' => '6.31'
  },
  'DISTNAME' => 'Task-BeLike-hanekomu',
  'EXE_FILES' => [],
  'LICENSE' => 'perl',
  'NAME' => 'Task::BeLike::hanekomu',
  'PREREQ_PM' => {
    'AnyEvent' => '0',
    'App::Ack' => '0',
    'App::Rgit' => '0',
    'App::cpanminus' => '0',
    'App::cpanoutdated' => '0',
    'App::distfind' => '0',
    'Coro' => '0',
    'DBD::SQLite' => '0',
    'DBI' => '0',
    'Devel::Loaded' => '0',
    'Devel::NYTProf' => '0',
    'Devel::SearchINC' => '0',
    'Dist::Zilla' => '0',
    'Dist::Zilla::Plugin::Git::Init' => '0',
    'Dist::Zilla::PluginBundle::MARCEL' => '0',
    'File::Slurp' => '0',
    'File::Which' => '0',
    'LWP' => '0',
    'Perl::Tidy' => '0',
    'Pod::Coverage::TrustPod' => '0',
    'Pod::Weaver::PluginBundle::MARCEL' => '0',
    'Pod::Wordlist::hanekomu' => '0',
    'Task::Plack' => '0',
    'Test::Differences' => '0',
    'Web::Scraper' => '0',
    'YAML' => '0'
  },
  'VERSION' => '1.103460',
  'test' => {
    'TESTS' => 't/*.t'
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



